### ISSUE TITLE
Merge all jump strategies into single one

### ISSUE THREAD

**Issue by isc30** [msomeone/PeasyMotion#18]
InvokePeasyMotionTwoCharJump is awesome, yet doesn't work when the character is at the end of the line.
It would be awesome to also have a single character version of it.

It would work in the following way:
- Ctrl+Enter are pressed
- Normaly PeasyMotion is invoked, and the first character from the "selector" matches the one you are selecting.
- If a key from the selector is pressed, it follows normal approach: substract chars from selector until last one, and then select.
- If a different key is pressed, find that char in all viewport and label the results. Following keypressed are used to select the new selectors.

**Note:** if the selected char is at the end of the line, it puts the cursor at the end of the line instead.

This would keep peasymotion simple but with flexibility to fully select whatever you need.
Also labels would be somehow predictable which improves speed.

**Comment by mm6502** [comment]
I'd like it to work just like the [FindJump for VS Code](https://marketplace.visualstudio.com/items?itemName=mksafi.find-jump)...

**Comment by msomeone** [comment]
Single character version is here. But unfortunately no plans to merge to single jump-strategy or do something similar to FindJump. Maybe FindJump can be implemented as a separate extension. @mm6502  @Sulfur-CK @samaursa Does FindJump behavior matches what @isc30 described?

**Comment by mm6502** [comment]
Basically, yes. It narrows down the possible jump locations as you type the word you are trying to jump to.

**Comment by Sulfur-CK** [comment]
@msomeone yes, FindJump behavior is exactly what I am hoping for. It's quite natural to type a few letters of the word/symbol. With PeasyMotion 2 letter jump, I routinely type in one too many letters and have to restart the search.
